app/src/main/java/com/example/next_show/data/RecommendationClient.java

```diff
@@ -29,20 +30,26 @@
 
     // instance vars
     private Context context;
+    private User currentUser;
 
     // empty constructor
     public RecommendationClient() {
 
     }
 
-    public RecommendationClient(Context context) {
+    public RecommendationClient(Context context, User currentUser) {
         this.context = context;
+        this.currentUser = currentUser;
     }
 
     // TODO: will need to use User's liked saved shows for this one unless -> random choice?
-    public void fetchRelatedShows(Shows traktShows, Show compShow, ShowAdapter adapter) {
-        // use given show ID to generate related, recommended shows
-        String searchID = compShow.getId();
+    public void fetchRelatedShows(Shows traktShows, ShowAdapter adapter) {
+        // log statement
+        Log.i(TAG, "Fetching related shows from Trakt!");
+
+        // use given show ID to generate related, recommended shows -> randomize it!
+        List<String> savedShows = currentUser.getLikedSavedShows();
+        String searchID = savedShows.get(new Random().nextInt(savedShows.size()));;
```

Comment:
What's the usage of the random int here for?

Reply:
If there are multiple saved shows, I don't want to do multiple Trakt API calls to get related shows back to back so in order to pick one, I just randomly choose one of the user's liked shows and grab data from there